vonk/start.rst

```diff
@@ -27,7 +27,7 @@ For non Windows systems, or if you want to use Docker for Windows, please look a
     }
 
 .. important:: the next step assumes you have a .Net Core environment installed. If not, please `download and install
-  <https://www.microsoft.com/net/download/core#/runtime/>`_ **.NET Core Runtime 2.0.0** before you continue.
+  <https://www.microsoft.com/net/download/core#/runtime/>`_ **.NET Core Runtime 2.1.10 or higher** before you continue.
```

Comment:
We've arrived at 2.2. Would change to 'latest 2.2.x version'

Reply:
2.1 is the LTS version. I expect that larger organisations will be stuck to 2.1 for a while. Should we really force the upgrade?

Reply:
We already did upgrade. Your comment is right, but too late I'm afraid.

Reply:
Oh, right. I was still thinking about the last Vonk version not about the new one.